src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -65,8 +66,9 @@ const TableHeader = ({ seatedAtTable, setSeatedAtTable, venueName }: any) => {
   // useWindowUnloadEffect(() => leaveSeat(), true);
 
   const leaveSeat = useCallback(async () => {
+    if (!user || !profile) return;
     const doc = `users/${user.uid}`;
-    const existingData = user.data;
+    const existingData = profile.data;
```

Comment:
They were both `user.smth` before and now it's split between `user` and `profile`. Was it broken before?

Reply:
there are two types of user data. first is from firebase auth (image 1) and second is from the user table (image 2) of the database. I am storing both in redux and my hook returns them separately as user and profile

![Screenshot 2020-07-28 at 13 18 36](https://user-images.githubusercontent.com/23658176/88664469-0644b900-d0d5-11ea-9f98-d8c81d4e8fc7.png)
![Screenshot 2020-07-28 at 13 18 45](https://user-images.githubusercontent.com/23658176/88664478-08a71300-d0d5-11ea-97d6-8fa57d43b0d3.png)